testing/src/main/java/com/firebase/geofire/testing/GeoFireTestingRule.java

```diff
@@ -47,28 +63,38 @@ public void starting(Description description) {
             }
 
             FirebaseOptions firebaseOptions = new FirebaseOptions.Builder()
-                    .setDatabaseUrl(DATABASE_URL)
+                    .setDatabaseUrl(databaseUrl)
                     .setCredentials(credentials)
                     .build();
             FirebaseApp.initializeApp(firebaseOptions);
 
             System.setProperty(SimpleLogger.DEFAULT_LOG_LEVEL_KEY, "DEBUG");
         }
-        this.databaseReference = FirebaseDatabase.getInstance().getReferenceFromUrl(DATABASE_URL);
+        this.databaseReference = FirebaseDatabase.getInstance().getReferenceFromUrl(databaseUrl);
     }
 
+    /** This will return you a new Geofire instance that can be used for testing. */
     public GeoFire newTestGeoFire() {
         return new GeoFire(databaseReference.child(randomAlphaNumericString(16)));
     }
 
+    /**
+     * Sets a given location key from the latitude and longitude on the provided Geofire instance.
+     * This operation will run asychronously.
+     */
     public void setLoc(GeoFire geoFire, String key, double latitude, double longitude) {
```

Comment:
If you wanna change these names, go for it!  Your suggestions SGTM

Reply:
I'd change them in a separate PR to only have the documentation changes here. Otherwise things get too mixed up in my opinion.